src/tests/gitpod/gitpodIdManager.test.ts

```diff
@@ -1,5 +1,5 @@
 import * as assert from 'assert';
-import * as mock from 'mock-fs';
```

Comment:
is mock-fs still used elswhere?  If not package.json+lock should be updated to remove it

Reply:
yes it is, well in a disabled test but required at compilation https://github.com/redhat-developer/vscode-redhat-telemetry/blob/7aa91a44721969edb223b1dee46625a0410de075/src/tests/services/configurationManager.disabled.ts#L4